app/src/main/java/com/github/gotify/messages/MessagesActivity.java

```diff
@@ -178,6 +180,11 @@ protected void onUpdateApps(List<Application> applications) {
 
     private Picasso makePicasso() {
         OkHttpClient.Builder builder = new OkHttpClient.Builder();
+        int CACHE_SIZE = 10 * 1024 * 1024; // 10 MB
+        builder.cache(
+                new Cache(
+                        new File(getApplicationContext().getCacheDir(), "http-cache"), CACHE_SIZE));
```

Comment:
It would be great, if the cache directory would be cleaned when the refresh button is pressed `com.github.gotify.messages.MessagesActivity#onRefreshAll` or when the messages are refreshed `com.github.gotify.messages.MessagesActivity#onRefresh`.

Reply:
Good idea. I opted for the onRefreshAll call as that made the most sense to me.